printer/what.sh

```diff
@@ -25,17 +25,17 @@ sleep 10
 
 # use jq to parse various config.json variables
 
-LEFT_PRINTER_NAME=$(jq -r '.PRINTER_LEFT_NAME' config/config.json)
-LEFT_PRINTER_IP=$(jq -r '.PRINTER_LEFT_IP' config/config.json)
+export LEFT_PRINTER_NAME=$(jq -r '.PRINTER_LEFT_NAME' config/config.json)
+export LEFT_PRINTER_IP=$(jq -r '.PRINTER_LEFT_IP' config/config.json)
 
-RIGHT_PRINTER_NAME=$(jq -r '.PRINTER_RIGHT_NAME' config/config.json)
-RIGHT_PRINTER_IP=$(jq -r '.PRINTER_RIGHT_IP' config/config.json)
+export RIGHT_PRINTER_NAME=$(jq -r '.PRINTER_RIGHT_NAME' config/config.json)
+export RIGHT_PRINTER_IP=$(jq -r '.PRINTER_RIGHT_IP' config/config.json)
 
 # we set these environment variables as the AWS S3 node library will
 # throw errors otherwise.
-AWS_ACCESS_KEY_ID=$(jq -r '.ACCESS_ID' config/config.json)${AWS_ACCESS_KEY_ID}
-AWS_SECRET_ACCESS_KEY=$(jq -r '.SECRET_KEY' config/config.json)${AWS_SECRET_ACCESS_KEY}
-AWS_DEFAULT_REGION=$(jq -r '.AWS_DEFAULT_REGION' config/config.json)${AWS_DEFAULT_REGION}
+export AWS_ACCESS_KEY_ID=$(jq -r '.ACCESS_ID' config/config.json)${AWS_ACCESS_KEY_ID}
+export AWS_SECRET_ACCESS_KEY=$(jq -r '.SECRET_KEY' config/config.json)${AWS_SECRET_ACCESS_KEY}
+export AWS_DEFAULT_REGION=$(jq -r '.AWS_DEFAULT_REGION' config/config.json)${AWS_DEFAULT_REGION}
```

Comment:
idk why we had the variables at the end of this line
```suggestion
export AWS_ACCESS_KEY_ID=$(jq -r '.ACCESS_ID' config/config.json)
export AWS_SECRET_ACCESS_KEY=$(jq -r '.SECRET_KEY' config/config.json)
export AWS_DEFAULT_REGION=$(jq -r '.AWS_DEFAULT_REGION' config/config.json)
```

Reply:
still do this